Modules/_decimal/_decimal.c

```diff
@@ -4348,18 +4339,9 @@ dec_##MPDFUNC(PyObject *self, PyObject *args, PyObject *kwds)             \
 
 /* Unary function with an optional context arg. */
```

Comment:
To improve clarity over what arguments are expected in the function definition:
```suggestion
/* Unary function with an optional context arg.
   Argument Clinic provides PyObject *self, PyObject *context
*/
```